packages/block-library/src/cover/style.scss

```diff
@@ -92,7 +92,7 @@
 	}
 
 	.wp-block-cover__inner-container {
-		width: calc(100% - 70px);
+		width: calc(100% - 4.375em);
```

Comment:
I think this one is maybe something to revert. I don't 100% fully understand where those 70px come from (cc: @jorgefilipecosta perhaps?) but I think given it's part of a `calc` statement, it was potentially important that this remained a pixel value.

Reply:
I reviewed the PR you submitted it seems we can just remove the rule 👍

Reply:
@jasmussen @jorgefilipecosta Please see comment here: 
https://github.com/WordPress/gutenberg/pull/25103#issuecomment-708611178

This is causing styling breakages on the frontend for cover blocks.

Reply:
Thanks again @johnstonphilip. I believe that #26143 to fix it.